backend/testfiles/execution/language.tests

```diff
@@ -129,6 +129,9 @@ List.push_v0 [] (fun x -> -4.611686018e+18) = [(fun x -> -4.611686018e+18)]
  let x = 6 in
  [1; 2; 3; 4] |> List.map_v0 y) = [ 6; 7; 8; 9 ]
 
+[test.pipe identity function]
+([1; 2; 3] |> List.map_v0 Fun.identity) = [1; 2; 3]
```

Comment:
This is apparently illegal. @pbiggar could you confirm that this _should_ be legal Dark code?

Reply:
It definitely isn't legal now, but it's also definitely an obvious thing that should be enabled/allowed.